deploy/charts/jetstack-agent/templates/configmap.yaml

```diff
@@ -6,8 +7,12 @@ metadata:
   labels:
     {{- include "jetstack-agent.labels" . | nindent 4 }}
 data:
+{{- if .Values.config.override.config }}
```

Comment:
This could just be a nullity check for example (cf suggestion on the values README)